smokefree/sites/all/themes/smokefreegov/templates/html.tpl.php

```diff
@@ -143,11 +143,16 @@ function f(n){if(c){var i=c.indexOf(n+'=');if(i>-1){var j=c.
   <!-- Facebook Open Graph Meta Tags  note: og:image is created in template.php -->
   <meta property="og:site_name" content="<?php if (isset($head_title_array['name'])) { echo $head_title_array['name']; } ?>" />
   <meta property="og:title"     content="<?php if (isset($head_title_array['title'])) { echo $head_title_array['title']; } ?>" />
-
   <?php if ($is_front) : ?>
     <meta property="og:image" content="<?php echo "https://".$_SERVER['HTTP_HOST']."/" . path_to_theme(); ?>/images/header/facebook-smokefree.jpg" />
+    <meta property="og:description" content="Smokefree.gov is an initiative from the National Cancer Institute to help you or someone you care about quit smoking." />
   <?php endif; ?>
 
+  <?php
+    // @TODO Remove this once we move the veterans pages into its own domain
+    if (request_path() == 'veterans') : ?>
+      <meta property="og:description" content="SmokefreeVET’s mission is to help veterans who get their health care through VA become tobacco-free." />
```

Comment:
Why can't be done using the metatag module, editing the veterans home homepage?

Reply:
The "/veterans" page is constructed using panels. I tried to see if there was a way to do this using the Drupal UI but it does not look like there is. 

There might be a module which might do what we need, but it looks like overkill for this task.

Reply:
EDIT: The metatag module comes with a "Metatag OG" and "Metatag Context" modules. We should have used the Metatag OG module to add the "og" tags as opposed to hardcoding them in the template.

The metatag content module lets you add custom metatag configurations by path. I am going to enable both the modules so we can add these settings the recommended way going forward.